tempodb/tempodb.go

```diff
@@ -382,6 +382,11 @@ func (rw *readerWriter) pollBlocklist() {
 		if err != nil {
 			metricBlocklistErrors.WithLabelValues(tenantID).Inc()
 			level.Error(rw.logger).Log("msg", "error polling blocklist", "tenantID", tenantID, "err", err)
+			rw.blockListsMtx.Lock()
+			delete(rw.blockLists, tenantID)
+			delete(rw.compactedBlockLists, tenantID)
+			rw.blockListsMtx.Unlock()
+			level.Error(rw.logger).Log("msg", "deleted in-memory blocklists", "tenantID", tenantID)
```

Comment:
The message sounds like a warning, if its error, it should detail the error, if performing a protective action, log as a warning.

Reply:
let's just go for info.  there's nothing error or warning about clearing a blocklist.  this will be expected behavior.